client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -251,6 +251,22 @@ const getAddressErrors = ( addressData, countriesData ) => {
 		}
 	}
 
+	if ( shouldValidatePhone ) {
+		// EasyPost requires an origin phone number for international shipments.
+		// This validation ensures that EasyPost will accept it, even though the phone may be invalid.
+		if ( ! phone ) {
+			errors.phone = translate( 'Origin phone number is required for international shipments' );
+		} else if (
+			10 !==
+			phone
+				.split( /\D+/g )
+				.join( '' )
+				.replace( /^1/, '' ).length
```

Comment:
this is for US phone numbers only, right?

note that we already depend on [phone](https://www.npmjs.com/package/phone) which can help

```js
phone('(817) 569-8900'); // return ['+18175698900, 'USA']
```

Reply:
> this is for US phone numbers only, right?

Yes, US and US territories.

> note that we already depend on phone which can help

I saw that, but it's a big dependency (right now it's not included in our `woocommerce-services` plugin build), for a very limited use.

There's a bit more context about this logic here: https://github.com/Automattic/woocommerce-connect-server/pull/1122
Basically, we want to use the same rudimentary validation EasyPost uses. In our experience, merchants generally don't care about phone numbers, so this is just to comply with a requirement by the platform.

Reply:
> we want to use the same rudimentary validation EasyPost uses

that's a really valid point but definitely worth an explanatory comment. someone will come around to this validation and think it's wrong

> but it's a big dependency

do you mean that we're importing this file from inside WooCommerce Services? I meant that Calypso already includes the dependency.